pkg/plugin/keyvault_test.go

```diff
@@ -63,13 +70,25 @@ func TestNewKeyVaultClient(t *testing.T) {
 			vaultName:   "\"testkv\"",
 			keyName:     "\"key1\"",
 			keyVersion:  "\"262067a9e8ba401aa8a746c5f1a7e147\"",
+			proxyMode:   false,
 			expectedErr: false,
 		},
+		{
+			desc:         "no error with proxy mode",
+			config:       &config.AzureConfig{ClientID: "clientid", ClientSecret: "clientsecret"},
+			vaultName:    "testkv",
+			keyName:      "key1",
+			keyVersion:   "262067a9e8ba401aa8a746c5f1a7e147",
+			proxyMode:    true,
+			proxyAddress: "localhost",
+			proxyPort:    7788,
+			expectedErr:  false,
+		},
```

Comment:
Could we also add a new `expectedVaultURL` var to the test struct and validate the url in the tests? It will be good to check the expected vault URL when proxy is enabled.

Reply:
fixed